omod/src/main/resources/webModuleApplicationContext.xml

```diff
@@ -20,9 +20,4 @@
   		    http://www.springframework.org/schema/util
   		    http://www.springframework.org/schema/util/spring-util-3.0.xsd">
 
-	<context:component-scan base-package="org.openmrs.module.tag.web.controller" />
-
-	<bean class="org.openmrs.ui.framework.StandardModuleUiConfiguration">
```

Comment:
If you will add them back, then you might as well keep them since they are needed anyways